tests/fs/test_idbfs_defer_write_on_exit.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <emscripten.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <errno.h>
+#include <string.h>
+
+int result = 1;
+
+int main() {
+  struct stat st;
+  int fd;
+
+  // We first open and close an empty file in the directory and leave, which
+  // should trigger syncfs() on exit. We will then verify the existance of the
+  // file on the !FIRST run.
```

Comment:
BTW this test also rely on the fact syncfs can't possibly spend for more than 1 sec (the time between we `REPORT_RESULT();` and we call `window.close()` in `tests/runner.py`).